docs/pricing.md

```diff
@@ -5,29 +5,32 @@
 
 Kodular Creator is **completely free of charge**. You can use it for whatever you want, and _we will never ask you to pay us_.
 
-We wanted it like that so **anyone could convert their desired ideas into apps** in a simple way. We think that _everybody should have the right to create apps even without knowing how to code_. That is why our platform is free and **it will remain like that forever**.
+We wanted it to be like that so **anyone could convert their awesome ideas into apps** in a simple way. We believe that _everybody should have the right to create apps even without knowing how to code_. That's why, our platform is free and **it will remain like that forever**.
 
 This also extends to **Kodular Store** and **Kodular Extensions IDE**.
 
 !!! quote "Why this business model?"
-    Kodular was created to providen accessible suite for everyone to learn and, later, professionally create apps. Due to this, we strongly believe learning to code has to be a right and nobody should have to pay in order to learn the basics of programming.  
-    Kodular Creator provides one of the most complete tools in Internet to create apps without coding, and moreover it is being used by tons of students all over the world. Everybody has to be able to make their idea reality independently of their income. Someone can have the next big revolutionary idea, but sometimes without money it cannot be done.  
+    Kodular was created to provide an accessible suite for everyone to learn and, later, professionally create apps. Due to this, we strongly believe learning to code has to be a right and nobody should have to pay in order to learn the basics of programming.  
+    Kodular Creator provides one of the most complete tools in Internet to create apps without coding, and moreover it is being used by tons of developers from all over the world. Everybody should be able to make their ideas a reality, independently of their income. Someone can have the next big revolutionary idea, but sometimes without money it cannot be done.  
```

Comment:
```suggestion
    Kodular Creator provides one of the most complete tools in Internet to create apps without coding, and moreover it is being used by tons of students from all over the world. Everybody should be able to make their ideas a reality, independently of their income. Someone can have the next big revolutionary idea, but sometimes without money it cannot be done.  
```

Keep students

Reply:
IMO, developers fits there better. Kodular is for everyone wanting to develop apps, and hence, isn't limited to the students.

Reply:
@barreeeiroo

Reply:
I've already committed other suggestions proposed by you. Any thoughts on this one?